src/glabs-left/web/index.js

```diff
@@ -0,0 +1,8 @@
+import { getIframeId, getWebfonts, resizeIframeHeight, reportClicks } from
+    '../../_shared/js/messages.js';
+import { write } from '../../_shared/js/dom.js';
+
+reportClicks();
+getIframeId()
+.then(() => getWebfonts())
+.then(() => resizeIframeHeight());
```

Comment:
Looks better! I'll adopt :) I think I'll leave the promise chaining as-is since (to me) it is more immediately obvious that there are three operations chained together.